hackathon/dat/banking_tool_calling_evaluation.py

```
@@ -0,0 +1,465 @@
+                
+                print(f"📊 Tool Calling Accuracy Metrics:")
+                report_data["detailed_metrics"]["tool_calling_accuracy"] = {}
+                
+                total_score = 0
+                metric_count = 0
+                
+                for score_name, score_result in metric.scores.items():
+                    score_value = score_result.value
+                    total_score += score_value
+                    metric_count += 1
+                    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)